.github/workflows/test-evm.yaml

```diff
@@ -119,10 +119,10 @@ jobs:
         run: node evm/deploy-its.js -m create2 -u -y
```

Comment:
Update orchestration scripts and merge together with this PR to avoid breaking it